src/components/my-career/SkillsRecommendationCourses.jsx

```diff
@@ -0,0 +1,130 @@
+import SearchCourseCard from '../search/SearchCourseCard';
+
+import { useDefaultSearchFilters, useSearchCatalogs } from '../search/data/hooks';
+import { UserSubsidyContext } from '../enterprise-user-subsidy';
+import { SubsidyRequestsContext } from '../enterprise-subsidy-requests';
+
+const SkillsRecommendationCourses = ({ index, subCategoryName, facetSkills }) => {
+  const { enterpriseConfig } = useContext(AppContext);
+  const {
+    subscriptionPlan, subscriptionLicense, couponCodes: { couponCodes }, enterpriseOffers,
+  } = useContext(UserSubsidyContext);
+  const { catalogsForSubsidyRequests } = useContext(SubsidyRequestsContext);
+
+  const searchCatalogs = useSearchCatalogs({
+    subscriptionPlan,
+    subscriptionLicense,
+    couponCodes,
+    enterpriseOffers,
+    catalogsForSubsidyRequests,
+  });
+
+  const { filters } = useDefaultSearchFilters({
+    enterpriseConfig,
+    searchCatalogs,
+  });
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [courses, setCourses] = useState([]);
+  const [hitCount, setHitCount] = useState(undefined);
+  const { refinements } = useContext(SearchContext);
+  const { skill_names: skills } = refinements;
+
+  const skillsFacetFilter = useMemo(
+    () => {
+      if (facetSkills) {
+        return facetSkills.map((skill) => `skill_names:${skill}`);
+      }
+      return [];
+    },
+    [facetSkills],
+  );
+  const moreCoursesLink = useMemo(
+    () => {
+      if (facetSkills) {
+        return `/${enterpriseConfig.slug}/search?showAll=1&content_type=course&skill_names=${facetSkills.join('&skill_names=')}`;
+      }
+      return null;
+    },
+    [enterpriseConfig, facetSkills],
+  );
+
+  useEffect(
+    () => {
+      async function fetchCourses() {
+        setIsLoading(true);
+        const { hits, nbHits } = await index.search('', {
+          filters: `content_type:course AND ${filters}`, // eslint-disable-line object-shorthand
+          facetFilters: [
+            skillsFacetFilter,
+          ],
+        });
+        if (nbHits > 0) {
+          const hitsCamelCased = camelCaseObject(hits);
+
+          setCourses(hitsCamelCased.length <= 3 ? hitsCamelCased
+            : hitsCamelCased.slice(0, 3));
+          setHitCount(nbHits);
+          setIsLoading(false);
+        } else {
+          setHitCount(nbHits);
+          setIsLoading(false);
+        }
+      }
+      fetchCourses();
+    },
+    [filters, index, skills, skillsFacetFilter],
+  );
+
+  if (hitCount === 0) {
+    return null;
+  }
+
+  return (
+    <div>
+      <h5 className="mb-3 mt-n4">More courses that teach you {subCategoryName} Skills</h5>
+      <CardGrid>
+        {courses.map(course => (
+          <SearchCourseCard
+            key={uuidv4()}
+            isLoading={isLoading}
+            hit={course}
+          />
+        ))}
+      </CardGrid>
+      <Hyperlink
+        className="mt-3"
+        onClick={() => {
+          window.location.href = moreCoursesLink;
```

Comment:
This is not a recommended way to redirect to a new url. See if we can use `history.push(`/${enterpriseConfig.slug}/search?<skill-names>`);`